app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    for part in command.split(" "):
+        if part[1] != part[2]:
```

Comment:
The condition `if part[1] != part[2]:` is incorrect. You should compare the second and third elements of the split command (i.e., the source and destination file names) to ensure they are not the same. The current logic will not work as intended.